src/web/app/components/dialogs/UpdateFontConvert.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Button } from 'antd';
+
+import Alert from 'app/actions/alert-caller';
+import AlertConstants from 'app/constants/alert-constants';
+// import browser from 'implementations/browser';
+import i18n from 'helpers/i18n';
+
+import styles from './UpdateFontConvert.module.scss';
+
+const LANG = i18n.lang.beambox.popup.text_to_path;
```

Comment:
Prefer to put the variable into function to update language dynamically and release memory after dialog popped